packages/next-swc/crates/next-core/src/app_source.rs

```diff
@@ -931,7 +931,6 @@ import {}, {{ chunks as {} }} from "COMPONENT_{}";
             context.compile_time_info().environment(),
         )
         .layer("ssr")
-        .css_chunk_root_path(server_root.join("_next/static/chunks"))
```

Comment:
This change will probably lead to CSS chunks being emitted into the output filesystem but nevermind

Reply:
Yeah. I still don't understand why the root path matters for invalidation purposes, but I don't think this is a problem.